### DISCUSSION TITLE
希望支持 Radxa E20C

### DISCUSSION THREAD

**Discussion by llhdxnq** [immortalwrt/immortalwrt#1639] [General]
希望可以增加 Radxa E20C这个设备

**Comment by yasuisss** [comment]
e52c希望也支持一波 谢谢

**Comment by symalgc** [comment]
e52c希望也支持一波 谢谢

**Comment by symalgc** [comment]
e52c用rock5a的没网络，提示
[   14.132488] rk_gmac-dwmac fe1c0000.ethernet eth0: Register MEM_TYPE_PAGE_POOL RxQ-0
[   14.133536] rk_gmac-dwmac fe1c0000.ethernet eth0: Register MEM_TYPE_PAGE_POOL RxQ-1
[   14.883064] RTL8211F Gigabit Ethernet stmmac-0:01: phy_poll_reset failed: -110
[   14.923909] rk_gmac-dwmac fe1c0000.ethernet eth0: __stmmac_open: Cannot attach to PHY (error: -110)
[   14.925716] br-lan: port 1(eth0) entered blocking state
[   14.926178] br-lan: port 1(eth0) entered disabled state
[   14.926652] rk_gmac-dwmac fe1c0000.ethernet eth0: entered allmulticast mode
[   14.927337] rk_gmac-dwmac fe1c0000.ethernet eth0: entered promiscuous mode

**Comment by 1715173329** [comment]
已支持。
